### PR TITLE
✨ feat: webhooks

### PR DESCRIPTION
**Pull request by deleteLater**
Allow users to subscribe to flag and segment changes

### CODE REVIEW

**Comment by deleteLater** [comment]
There will have some other webhook features be done in other PRs

- add human-readable changes description to our webhook payload
- view webhook logs

File: modules/front-end/src/app/core/services/handlebars.service.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+@Injectable({
+  providedIn: 'root',
+})
+
+export class HandlebarsService {
+
+  private _initialized = false;
+
+  init() {
+    if (this._initialized) {
```
Comment (cosmos-explorer) [inline modules/front-end/src/app/core/services/handlebars.service.ts:14]: Can't we put this logic in constructor? Services are singletons

File: modules/front-end/src/app/core/services/monaco-service.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+@Injectable({
+  providedIn: 'root',
+})
+
+export class MonacoService {
+
+  private _initialized = false;
+
+  init() {
+    if (this._initialized) {
```
Comment (cosmos-explorer) [inline modules/front-end/src/app/core/services/monaco-service.ts:15]: Can't we put this logic in constructor? Services are singletons

**Comment by cosmos-explorer** [comment]
image
Maybe use another separator here?

**Review by cosmos-explorer — APPROVED** [review]
LGTM

**Pull request merged**